### PR TITLE
[8] feat: add stats svg

### LINKED ISSUE #8: Dynamic SVG for stats

### PR DESCRIPTION
**Pull request by pjotrvdh**
seed and go to https://dlf-hackathon-2023.swisdev.nl/stats/{tenant_id} and see a stats svg

### CODE REVIEW

**Comment by JaZo** [comment]
:information_source: Now with type hints in your code, defining types within PHP DocBlocks is redundant. Laravel has removed all of the `@param` and `@return` tags from its DocBlocks where types are defined with PHP. Similarly, Shift [removed these tags](https://github.com/swisnl/dlf-hackathon-2023/pull/25/commits/341eed3158e8905415740743244877a270c8a2d5?w=1) from any DocBlock where the code now has _equivalent_ type hints.

**Comment by JaZo** [comment]
:information_source: While the _Laravel Fixer_ performs automation to adopt the latest Laravel conventions, Shift understands developers have different preferences. All of Shift's automation is done in nice, atomic commits. This makes it easier to undo any of the changes Shift makes using `git revert`.

File: routes/web.php [hunk 1]
```diff
@@ -20,3 +21,13 @@
 Route::get('/transactions/create', [PaymentController::class, 'create'])->name('transactions.create');
 Route::get('/transactions/success', [PaymentController::class, 'success'])->name('transactions.redirect');
 Route::post('/transactions/webhook', [PaymentController::class, 'webhook'])->name('transactions.webhook');
+
+Route::get('/stats/{tenant}', static function (Tenant $tenant) {
```
Comment (AnthonySchuijlenburg) [inline routes/web.php:25]: Tenants hebben specifiek hiervoor een slug, zou je die kunnen gebruiken?

**Review by AnthonySchuijlenburg — CHANGES_REQUESTED** [review]

File: routes/web.php (hunk 1 above)
Comment (pjotrvdh) [reply routes/web.php:25]: We've chosen to use the uuid instead of the slug

**Comment by Rocksheep** [comment]
The fonts are now a bit random, can you have a look at them?

**Review by AnthonySchuijlenburg — APPROVED** [review]

**Pull request merged**